test/dialog-flow/directives.spec.ts

```diff
@@ -824,4 +824,48 @@ describe("DialogFlow Directives", () => {
     });
   });
 
+  describe("ImmersiveResponse", () => {
+    it("should include a new immersive response directive", async () => {
+      const dialogFlowImmersiveResponse = {
+        loadImmersiveUrl: "https://www.example.com/path",
+        updatedState: {
+          totalMinutes: 10,
+        },
+      };
+
+      app.onIntent("LaunchIntent", {
+        dialogFlowImmersiveResponse,
+        flow: "yield",
+        sayp: "Hello!",
```

Comment:
Is this a typo?

Reply:
No, the voxa v3 has this property to assign text directly. Text, no path to a view

Reply:
Oh nice, I didn't know.